README.md

```diff
@@ -68,53 +29,34 @@ After you're finished please remove all the comments and instructions!
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
-  <summary>Client</summary>
+  <summary>visual studio code</summary>
   <ul>
     <li><a href="https://reactjs.org/">React.js</a></li>
```

Comment:
- [ ] Always use the same pull request for the same project until you get approved. in case of net time, you're meant to use the https://github.com/igna10697/Hello/pull/1 again to ask for this second review. if you read the review summary from the previous review, you would have seen such. It is also pointed out in my review below. please always read them completely.